web/src/services/AppointmentService.ts

```diff
@@ -5,13 +5,13 @@ const API_URL = 'http://127.0.0.1:8080/appointments';
 
 async function mapResponse(response: AxiosResponse): Promise<AppointmentSchema[]> {
     return response.data.map((item: any) => ({
-        user: item.id? item.id : -1,
-        appointmentType: item.appointmentType? item.appointmentType : "N/A",
+        user: item.user? item.user.name : -1,
```

Comment:
campo não refere-se a um número então não faz sentido utilizarmos o valor `-1` como default
```suggestion
        user: item.user? item.user.name : "N/A",
```